doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -0,0 +1,83 @@
+\subsubsection{Phase diagram}
+
+\label{sec:phase-diagram}
+For many models it is useful to see how the material parameters vary as a function of pressure and temperature. In this cookbook, a plot of material properties (e.g., density or viscosity) as a function of temperature (x axis) and pressure (y axis) is referred to as a phase diagram. 
+
+In this phase diagram cookbook, phase diagrams are generated from model runs and visualized with VISIT, but Paraview could also be used to view the same output. % intro
+This is a simple way of using the visualization output to check the correct implementation of phase transitions in a material model. 
+The idea lies in using a model with an initial temperature that increases linearly along the x-axis and a pressure that changes linearly along the y-axis in a box geometry, so that the two axes, x and y, can be used as axes for pressure and temperature in the graphical output. 
+Here, we visualize a diagram of the phase transitions implemented in the visco plastic material model, as well as a lookup table in the Steinberger material model. 
+
+\paragraph{The input file.}
+You can find the input file to run this cookbook example in \url{cookbooks/phase_diagram.prm}. For this first case, phase transitions are prescribed manually in terms of their depth, Clapeyron slope, and other key parameters. 
+
+\par The model domain is  800 km  by 800 km box. % geometry
```

Comment:
Here and below, drop the `\par` commands -- let latex figure out the layout of text.
```suggestion
The model domain is  800 km  by 800 km box. % geometry
```